app.js

```diff
@@ -120,27 +121,33 @@ function refreshAll(all) {
    var githubPulls;
 
    if (all === true) {
+      debug("rebuilding all pulls");
```

Comment:
The debug module is what we use for debug output (throughout the whole project).

`DEBUG='*' bin/pulldasher` to see all output.